tests/playwright/sanity/modules/nested-accordion/helper.ts

```diff
@@ -99,3 +99,17 @@ export async function setIconColor( editor, state, color, context ) {
 		await editor.page.click( '.elementor-control-header_' + state + '_' + context );
 	}
 }
+
+export async function addIcon( editor, page, iconName ) {
```

Comment:
if you add types and there is typescript support in your IDE, so will all the errors and type mismatches. This is just a hint.